src/main/java/com/kodilla/ecommercee/repository/ProductRepository.java

```diff
@@ -10,6 +10,15 @@
 @Repository
 @Transactional
 public interface ProductRepository extends CrudRepository<Product, Long> {
+
+
     List<Product> findByProductName(String name);
+
     List<Product> findByProductId(Long id);
+
+
```

Comment:
gdyby tu była metoda
@Override
List<Product> findAll();
nie trzeba by było mapować w teście do listy produktów  (linijka 53 w klasie testów).
To można było przewidzieć wcześniej ale nie zauważyliśmy :) To nie jest błąd (chyba) ale zwiększyłoby czytelność kodu

Reply:
poprawiłem